test/e2e/autoscaling_test.go

```diff
@@ -23,38 +24,63 @@ func TestAutoscaling(t *testing.T) {
 	t.Parallel()
 	g := NewWithT(t)
 
+	if globalOpts.configurableClusterOptions.NodePoolReplicas != 1 {
+		t.Fatalf("this test must be run with --e2e.node-pool-replicas=1, is configured to run with %d", globalOpts.configurableClusterOptions.NodePoolReplicas)
```

Comment:
why this constraint?

Reply:
The test is hardcoded to set a lower boundary of one and upper boundary of two for autoscaling, it doesn't pass with any other setting